atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresGetRange.java

```diff
@@ -138,7 +139,8 @@ public PostgresGetRange(PostgresPrefixedTableNames prefixedTableNames,
                                                long timestamp) {
         int maxRowsPerPage = RangeHelpers.getMaxRowsPerPage(rangeRequest);
         int cellsPerRowEstimate = getCellsPerRowEstimate(tableRef, rangeRequest);
-        int maxCellsPerPage = maxRowsPerPage * cellsPerRowEstimate + 1;
+        int maxCellsPerPage = Math.min(
```

Comment:
I think logging at warn is only going to alarm users for a situation that should actually be better than what we had before.  Happy to not have the logging for now as I suspect there's a lot of things we're going to look at before we get to checking this.